### PR TITLE
test(ext/console): Compact empty iterables when calling Deno.inspect with compact false

### PR DESCRIPTION
**Pull request by bcheidemann**
When `Deno.inspect` is called with the `compact` option set to `false`, empty iterables will still be compacted.

**Before**
```ts
Deno.inspect([], { compact: false });
// [
// 
// ]

Deno.inspect([], { compact: false, trailingComma: true });
// [
//   ,
// ]
```

**Now**
```ts
Deno.inspect([], { compact: false });
// []

Deno.inspect([], { compact: false, trailingComma: true });
// []
```

This is required to fix [this issue in denoland/deno_std](https://github.com/denoland/deno_std/issues/2144).

### CODE REVIEW

**Comment by kt3k** [comment]
@bcheidemann 
Thanks for the PR. I see this fixes the issue, but ideal fix would be like the below in my view:

```js
Deno.inspect([], { compact: false, trailingComma: true });
// [
//   
// ]
```

(This has less effect to the existing code. Also this will produce better diff when comparing, for example, `[]` with `[1]` in snapshot testing)

What do you think?

**Comment by bcheidemann** [comment]
> @bcheidemann Thanks for the PR. I see this fixes the issue, but ideal fix would be like the below in my view:
> 
> ```js
> Deno.inspect([], { compact: false, trailingComma: true });
> // [
> //   
> // ]
> ```
> 
> (This has less effect to the existing code. Also this will produce better diff when comparing, for example, `[]` with `[1]` in snapshot testing)
> 
> What do you think?

I can see the benefit of this from both perspectives - better snapshot diffs and minimising impact on existing code.

However, I would suggest that the following would make more sense.

```js
Deno.inspect([], { compact: false, trailingComma: true });
// [
// ]
```

This way the diff would be:

```
Expected + / Actual -

   [
-    3
   ]
```

Instead of:

```
Expected + / Actual -

   [
+
-    3
   ]
```

Let me know which you would prefer @kt3k - happy to implement either 🙂

File: ext/console/02_console.js [hunk 1]
```diff
@@ -465,12 +465,18 @@
...
+      level,
+    );
+    const closingIndentation = `${
+      inspectOptions.trailingComma ? "," : ""
+    }\n${closingDelimIndentation}`;
 
     let iContent;
-    if (options.group && entries.length > MIN_GROUP_LENGTH) {
+    if (entries.length === 0 && !inspectOptions.compact) {
+      iContent = `\n${closingDelimIndentation}`;
```
Comment (kt3k) [inline ext/console/02_console.js:478]: 👍

**Review by kt3k — APPROVED** [review]
LGTM!

**Review by bartlomieju — APPROVED** [review]
LGTM

**Pull request merged**